test/test_jit.py

```diff
@@ -482,6 +482,18 @@ def test_relu(self):
 
         ge = self.checkTrace(self.fn_test_relu, (x, y))
 
+    @staticmethod
+    def fn_test_small_constant(x, y):
+        return (1e-8 * x + 5e-9 * y) * 1e8
```

Comment:
Why can't you put this function in the body of `test_small_constant`? That's what we usually do. This let us avoid leaking it to the surroundings.